src/Player/PlayerPrimitive.tsx

```diff
@@ -36,34 +36,44 @@ const PlayerPrimitive: React.FC<Props> = ({ controls, inline = false, playedAt,
     player.seekTo(start, 'seconds')
   }, [player, playedAt])
 
-  const Player = (
-    <ReactPlayer
-      ref={setRefFromPlayer}
-      controls={controls}
-      url={`https://www.youtube.com/watch?v=${youtubeId}&nonce=${playedAt}`}
-      playing={true}
-      progressInterval={500}
-      volume={unmutedPlayer === playerIdentifier ? (volume || 0) / 100.0 : 0}
-      height="350px"
-      width="100%"
-      style={{ border: '1px solid #2D3748' }}
-      onProgress={setProgress}
-    />
-  )
+  const setVideoDimensions = (ref: Element | undefined): void => {
+    if (!ref) {
+      return
+    }
 
-  if (inline) {
-    return Player
+    const { top, bottom, left, width } = ref.getBoundingClientRect()
+    const right = window.innerWidth - left - width
+    setDimensions({ top, right, bottom, left })
   }
 
-  if (!showVideo) {
-    return <Box sx={{ visibility: 'hidden', height: 0 }}>{Player}</Box>
-  }
+  useEffect(() => {
+    setVideoDimensions(videoRef?.current)
 
-  if (!!videoRef && !!videoRef.current) {
-    return createPortal(Player, videoRef.current)
-  }
+    const onResize = (): void => setVideoDimensions(videoRef?.current)
+    window.addEventListener('resize', onResize)
```

Comment:
@DLavin23 Whoops, refactored this a bit after I left the comment.  Here's where we fix the window resize bug that we were seeing.